src/SmartComponents/Inventory/applications/compliance/SystemPolicyCards.js

```diff
@@ -6,16 +6,27 @@ import propTypes from 'prop-types';
 import { Instagram } from 'react-content-loader';
 
 class SystemPolicyCards extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    systemPolicyCards() {
+        const policyCards = this.props.policies.map(
+            (policy, i) =>
+                <GridItem span={ 4 } key={ i }>
+                    <SystemPolicyCard policy={ policy } />
+                </GridItem>
+        );
+        return policyCards;
+    }
+
     render() {
-        const { policies, loading } = this.props;
+        const { loading } = this.props;
+
         return (
             <div id="system_policy_cards">
```

Comment:
Why are we using an ID here?

Reply:
QE testing

Reply:
Shouldn't it be a `widget-id` or a `widget-type` then?